tests/suites/test_suite_psa_its.data

```diff
@@ -0,0 +1,65 @@
+
+Set/get/remove with flags
+set_get_remove:0:0x12345678:"abcdef"
+
+Overwrite 0 -> 3
+set_overwrite:0:0x12345678:"":0x01020304:"abcdef"
+
+Overwrite 3 -> 0
+set_overwrite:0:0x12345678:"abcdef":0x01020304:""
+
+Overwrite 3 -> 3
+set_overwrite:0:0x12345678:"123456":0x01020304:"abcdef"
+
+Overwrite 3 -> 18
+set_overwrite:0:0x12345678:"abcdef":0x01020304:"404142434445464748494a4b4c4d4e4f5051"
+
+Overwrite 18 -> 3
+set_overwrite:0:0x12345678:"404142434445464748494a4b4c4d4e4f5051":0x01020304:"abcdef"
+
+Multiple files
+set_multiple:0:5
+
+Non-existent file
+nonexistent:0:0
+
+Removed file
+nonexistent:0:1
+
+Get 0 bytes of 10 at 10
+get_at:0:"40414243444546474849":10:0:PSA_ITS_SUCCESS
+
+Get 1 byte of 10 at 9
+get_at:0:"40414243444546474849":9:1:PSA_ITS_SUCCESS
+
+Get 0 bytes of 10 at 0
+get_at:0:"40414243444546474849":0:0:PSA_ITS_SUCCESS
+
+Get 1 byte of 10 at 0
+get_at:0:"40414243444546474849":1:0:PSA_ITS_SUCCESS
+
+Get 2 bytes of 10 at 1
+get_at:0:"40414243444546474849":1:2:PSA_ITS_SUCCESS
+
+Get 1 byte of 10 at 10: out of range
+get_at:0:"40414243444546474849":10:1:PSA_ITS_ERROR_INCORRECT_SIZE
+
+Get 1 byte of 10 at 11: out of range
+get_at:0:"40414243444546474849":11:1:PSA_ITS_ERROR_INCORRECT_SIZE
+
+Get 0 bytes of 10 at 11: out of range
+get_at:0:"40414243444546474849":11:1:PSA_ITS_ERROR_INCORRECT_SIZE
```

Comment:
The title of this test ("Get 0 bytes of 10 at 11: out of range") implies it should be different in content than "Get 1 byte of 10 at 11: out of range", but it looks like it's testing the same thing (in commit "Tests for PSA ITS over files").

```diff
- get_at:0:"40414243444546474849":11:1:PSA_ITS_ERROR_INCORRECT_SIZE
+ get_at:0:"40414243444546474849":11:0:PSA_ITS_ERROR_INCORRECT_SIZE
```